image_width: 1920
image_height: 1080
camera_name: ps5eye_right
camera_matrix:
  rows: 3
  cols: 3
  data: [1241.402020, 0.000000, 986.659635, 0.000000, 1240.630391, 564.448339, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.000045, -0.024059, 0.001207, 0.000039, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [1229.525757, 0.000000, 987.249202, 0.000000, 0.000000, 1238.181030, 564.865239, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
